code/modules/cooking/raw/NeuFood_dough.dm

```diff
@@ -1030,3 +1129,357 @@
+	desc = "A delightfully fancy treat adored by the upper echelons of Kingsfield, complete with tangerine frosting."
+	icon_state = "cookedtangerinescone"
+	list_reagents = list(/datum/reagent/consumable/nutriment = SNACK_DECENT+SNACK_POOR)
+	tastes = list("crumbly butterdough" = 1, "sweet" = 1, "tangerine" = 1)
+	faretype = FARE_FINE
+	eat_effect = /datum/status_effect/buff/foodbuff
+
+/*	.................   Plum Scone   ................... */
+
+/obj/item/reagent_containers/food/snacks/foodbase/scone_raw_plum
+	name = "unbaked plum scone"
+	icon_state = "uncookedplumscone"
+	cooked_type = /obj/item/reagent_containers/food/snacks/scone_plum
+	cooked_smell = /datum/pollutant/food/scone
+	rotprocess = SHELFLIFE_EXTREME
+	eat_effect = null
+
+/obj/item/reagent_containers/food/snacks/scone_plum
+	name = "plum scone"
+	desc = "A delightfully fancy treat adored by the upper echelons of Kingsfield, complete with plum filling."
+	icon_state = "cookedplumscone"
+	list_reagents = list(/datum/reagent/consumable/nutriment = SNACK_DECENT+SNACK_POOR)
+	tastes = list("crumbly butterdough" = 1, "sweet" = 1, "plum" = 1)
+	faretype = FARE_FINE
+	eat_effect = /datum/status_effect/buff/foodbuff
+
+/*-------------\
+| Griddlecakes |
+\-------------*/
+
+/*	.................   Griddlecakes   ................... */
+
+/obj/item/reagent_containers/food/snacks/foodbase/griddlecake_raw
+	name = "raw griddlecake"
+	icon_state = "rawgriddlecake"
+	fried_type = /obj/item/reagent_containers/food/snacks/griddlecake
+	cooked_smell = /datum/pollutant/food/griddlecake
+	rotprocess = SHELFLIFE_LONG
+	eat_effect = null
+
+/obj/item/reagent_containers/food/snacks/foodbase/griddlecake_raw/good
+	fried_type = /obj/item/reagent_containers/food/snacks/griddlecake/good
+
+/obj/item/reagent_containers/food/snacks/griddlecake
+	name = "griddlecake"
+	desc = "Enjoyed by soldiers of fortune all throughout Psydonia, though despite its prevelance no one quite knows its origin. This one looks a bit charred."
+	icon_state = "griddlecake"
+	list_reagents = list(/datum/reagent/consumable/nutriment = SNACK_DECENT+SNACK_POOR)
+	tastes = list("fluffy dough" = 1, "sweet" = 1)
+	rotprocess = SHELFLIFE_LONG
+	faretype = FARE_NEUTRAL
+
+/obj/item/reagent_containers/food/snacks/griddlecake/good
+	name = "perfect griddlecake"
+	desc = "Enjoyed by soldiers of fortune all throughout Psydonia, though despite its prevelance no one quite knows its origin."
+	faretype = FARE_FINE
+	eat_effect = /datum/status_effect/buff/foodbuff
+/obj/item/reagent_containers/food/snacks/griddlecake/good/New()
+	. = ..()
+	good_quality_descriptors()
+
+/obj/item/reagent_containers/food/snacks/griddlecake/good/attackby(obj/item/I, mob/living/user, params)
+	..()
+	if(user.mind)
+		short_cooktime = (50 - ((user.mind.get_skill_level(/datum/skill/craft/cooking))*8))
+	var/found_table = locate(/obj/structure/table) in (loc)
+	if(isturf(loc)&& (found_table))
+		if(istype(I, /obj/item/reagent_containers/food/snacks/butterslice))
```

Comment:
perhaps a skillcheck ? would be good to have more stuff for actual experienced chefs